docs/howto/edit-in-protege.md

```diff
@@ -0,0 +1,94 @@
+# Editing a term in protege
+
+Before you start:
+
+-   make sure you are working on a branch 
```

Comment:
Can we link this to a page that explains what branches are and/or how to create a branch?

Reply:
For now I'll put back the link that has slides on this, but I'd rather a more straightforward document (why I removed it initially)